ops_omgx/docker/go-ethereum/wait-for-l1.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
I'm a little bit confused what this service is.

Is this L1 Geth or Verifier?

Based on env vars below, it seems to be the verifier, but the path naming implies it's L1 Geth.

Reply:
The is the geth_L2 running in Sequencer mode. Right now, the optimism codes uses 5 different name for the L2, namely: l2geth, geth, geth_l2, go-ethereum, and l2-node. Assuming the top level folder naming is the definitive one, suggest we call this service the `l2geth`. So the correct name for this file/folder would be /docker/l2geth/ etc etc.